app/src/components/drawer/index.tsx

```diff
@@ -36,7 +36,9 @@ const Drawer: React.FC<Props> = ({ addWidgetToLayout }) => {
                 </Button>
               </div>
             )
-          )}
+          ) : 
+            <div className="flex justify-between py-2 italic">{t("widget.common.noWidget")}</div>
```

Comment:
I think we shouldn't use a category that has no widgets (that's why the unused are ones commented out). This way, this change (and the according label) are unnecessary. That said, the widget drawer requires [a major redesign](https://dashboard.darekkay.com/docs/roadmap/#features-and-improvements) anyway.